Documentation.md

```diff
@@ -192,562 +202,578 @@ code to be called when new agent information is available.
 * `name`: The name of the routing profile.
-* `queues`: The queues contained in the routing profile.
-* `defaultOutboundQueue`: The default queue which should be associated with outbound contacts.
+* `queues`: The queues contained in the routing profile. Each queue object has the following properties:
+  * `name`: The name of the queue.
+  * `queueARN`: The ARN of the queue.
+  * `queueId`: Alias for `queueARN`.
+* `routingProfileARN`: The routing profile ARN.
+* `routingProfileId`: Alias for `routingProfileARN`.
 
-### `agent.getChannelConcurrency`
-```
-if (agent.getChannelConcurrency("VOICE")) { ... }
-```
-OR
-```
-const concurrencyMap = agent.getChannelConcurrency();
+### `agent.getChannelConcurrency()`
+```js
+var concurrencyMap = agent.getChannelConcurrency(); // e.g. { VOICE: 0, CHAT: 2 }
+
+// OR
+
+if (agent.getChannelConcurrency("CHAT") > 1) { /* ... */ }
 ```
-Gets either a boolean represented by a 1 or 0, or a map of channel type to 1 or 0. 1 represents an enabled channel. 0 represents a disabled channel.
-* `channel`: A string ENUM representing the channel whose value to return. 
-The ENUM options are currently `"VOICE"` and `"CHAT"`
+Gets either a number or a map of `ChannelType`-to-number indicating how many concurrent contacts can an agent have on a given channel. 0 represents a disabled channel. The optional `channel` argument must be a value of the enum `ChannelType`.
 
 ### `agent.getName()`
-```
+```js
 var name = agent.getName();
 ```
-Gets the agent's user friendly display name from the `AgentConfiguration` object for the agent.
+Gets the agent's user friendly display name.
 
 ### `agent.getExtension()`
-```
+```js
 var extension = agent.getExtension();
 ```
-Gets the agent's phone number from the `AgentConfiguration` object for the agent.  This is the phone
-number that is dialed by Amazon Connect to connect calls to the agent for incoming and outgoing calls if
-softphone is not enabled.
+Gets the agent's phone number. This is the phone number that is dialed by Amazon Connect to connect calls to the agent for incoming and outgoing calls
+if softphone is not enabled.
 
 ### `agent.getDialableCountries`
-```
-if (agent.getDialableCountries().includes(COUNTRY_CODE)){ ... }
+```js
+var countries = agent.getDialableCountries(); // e.g. ["us", "ca"]
 ```
 Returns a list of eligible countries to be dialed / deskphone redirected.
 
 ### `agent.isSoftphoneEnabled()`
+```js
+if (agent.isSoftphoneEnabled()) { /* ... */ }
 ```
-if (agent.isSoftphoneEnabled()) { ... }
-```
-Determine if softphone is enabled for the agent.
+Indicates whether the agent's phone calls should route to the agent's browser-based softphone or the
+telephone number configured as the agent's extension.
 
 ### `agent.setConfiguration()`
-```
+```js
 var config = agent.getConfiguration();
-config.extension = "+12061231234";
+config.extension = "+18005550100";
```

Comment:
I though it would be more appropriate to use [fictional phone numbers](https://en.wikipedia.org/wiki/555_(telephone_number)#Fictional_usage).